packages/react-components/src/components/OnboardingChecklist/OnboardingChecklist.stories.tsx

```diff
@@ -0,0 +1,147 @@
+        </Button>
+        <Button kind="plain" onClick={() => secondaryHandler('2')}>
+          Skip
+        </Button>
+      </>
+    ),
+  },
+  {
+    id: '2',
+    title: 'Install LiveChat on your website',
+    description: 'Invite your team to help you with chats',
+    placeholder: <Icon size="xxxlarge" source={Icons.HelpDeskColored} />,
+    cta: (
+      <>
+        <Button
+          kind="high-contrast"
+          icon={<Icon source={Icons.OpenInNew} />}
+          iconPosition="right"
+          onClick={() => primaryHandler('2')}
+        >
+          Chat with yourself
+        </Button>
+      </>
+    ),
+  },
+  {
+    id: '3',
+    title: 'Invite your teammates',
+    description: 'Save time by using prepared answers',
+    placeholder: <Icon size="xxxlarge" source={Icons.ChatBotColored} />,
+    cta: (
+      <>
+        <Button
+          kind="high-contrast"
+          icon={<Icon source={Icons.OpenInNew} />}
+          iconPosition="right"
+          onClick={() => primaryHandler('3')}
+        >
+          Chat with yourself
+        </Button>
+      </>
+    ),
+  },
+  {
+    id: '4',
+    title: 'Set up a chatbot',
+    description: 'Collect feedback from your customers',
+    placeholder: <Icon size="xxxlarge" source={Icons.KnowledgeBaseColored} />,
+    cta: (
+      <>
+        <Button
+          kind="high-contrast"
+          icon={<Icon source={Icons.OpenInNew} />}
+          iconPosition="right"
+          onClick={() => primaryHandler('4')}
+        >
+          Chat with yourself
+        </Button>
+      </>
+    ),
+  },
+  {
+    id: '5',
+    title: 'Upload profile picture',
+    description: 'Automate your customer service',
+    placeholder: <Icon size="xxxlarge" source={Icons.OneColored} />,
+    cta: (
+      <>
+        <Button
+          kind="high-contrast"
+          icon={<Icon source={Icons.OpenInNew} />}
+          iconPosition="right"
+          onClick={() => primaryHandler('5')}
+        >
+          Chat with yourself
+        </Button>
+      </>
+    ),
+  },
+];
+
+export const Default = (): React.ReactElement => {
+  const [activeItem, setActiveItem] = React.useState<string>('1');
+  const [checkedIds, setCheckedIds] = React.useState<string[]>([]);
+  const [isCompleted, setIsCompleted] = React.useState<boolean>(false);
+
+  const handlePrimaryClick = (id: string) => {
+    setCheckedIds((prev) => [...prev, id]);
+    setActiveItem((prev) => (parseInt(prev) + 1).toString());
+
+    if (checkedIds.length === 4) {
+      setIsCompleted(true);
+    }
+  };
+
+  const handleSecondaryClick = (id: string) => setActiveItem(id);
+
+  return (
+    <OnboardingChecklist
+      title="Here’s your getting started guide"
+      titleLabel="Hello, Tim!"
+      items={getItems(handlePrimaryClick, handleSecondaryClick)}
+      activeId={activeItem}
+      checkedId={checkedIds}
+      onActiveChange={handleSecondaryClick}
+      isCompleted={isCompleted}
+      completeItem={{
```

Comment:
This name can be confusing - I'd suggest using something like: `completionMessageData`